From: Antonio Alvarez Feijoo <antonio.feijoo@suse.com>
Date: Mon, 7 Sep 2026 11:31:50 +0200
Subject: fix(dracut): export prefer_dracutmodules to be used in dracut
 modules

In the same way that their `*dracutmodules` siblings are exported.

Follow-up for 4b8c11c0590b546e37908968042eaf7c7d669b17
Origin: upstream, https://github.com/dracut-ng/dracut/commit/f2990cbabec2ea464cfb8531e4d38d5f8d138580
---
 dracut.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dracut.sh b/dracut.sh
index fa076aa..c892a52 100755
--- a/dracut.sh
+++ b/dracut.sh
@@ -2578,7 +2578,7 @@ done
 
 export initdir dracutbasedir \
     dracutmodules force_add_dracutmodules add_dracutmodules omit_dracutmodules \
-    mods_to_load \
+    prefer_dracutmodules mods_to_load \
     fw_dir drivers_dir debug no_kernel kernel_only \
     omit_drivers mdadmconf lvmconf root_devs \
     use_fstab fstab_lines libdirs fscks nofscks ro_mnt \
